### ISSUE TITLE
Open Calendar on specific date on init

### ISSUE THREAD

**Issue by peio72** [bradberger/angular-material-calendar#31]
Please could document how to open the calendar on a specific date (as it was clicked) on init ?

I tried using variables in data-start-month and data-start-year but not working.

Giving a value to  $scope.selectedDate not working too.

Thanx for your help.

**Comment by peio72** [comment]
I managed to switch calendar init display using: data-start-month="{{start_month}}" data-start-year="{{start_year}}".
Now i need to simulate the click on a day to open a selected day value.
Thanx.

**Comment by zilions** [comment]
@peio72 - I couldn't seem to get that working. My browser would always crash when adding in those two attributes... seems like some kind of infinite loop. I'm passing the month in as `2` (for Feb) and the year as `2015`. Any clue what's going wrong here?

**Comment by zilions** [comment]
Of course, silly me. Date indexes in Javascript start at 0 for Jan. Hence making Feb `0`.

**Comment by bradberger** [comment]
Hey guys, thanks for the patience on this one, I'm going to take a look at the issues in the near future, so we'll get it figured out. I do believe the start month/year attributes are just attributes, and not part of the scope, so that would mean passing just a value in, and not a variable. But for the endless loop issue, I'll check that out.

Either way we should definitely add documentation for this, too.

**Comment by remi-bruguier** [comment]
The infinite loop happens if you pass an object to the startMonth parameter. 
startMonth is not part of the angular-material-calendar directive scope, but is retrieved from the dom attribute:

```javascript
var month = parseInt($attrs.startMonth || date.getMonth());
```

The month variable is set to NaN in that case which provokes the loop. 
Just pass the value to your directive instead of the object and you should be fine:

```javascript
<calendar-md flex layout layout-fill
                 data-start-month="{{startMonth}}"
                  data-start-year="{{startYear}}"
                   ></calendar-md>
```
